src/main/java/com/cbfacademy/apiassessment/TreeController.java

```diff
@@ -0,0 +1,94 @@
+package com.cbfacademy.apiassessment;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/trees")
```

Comment:
You can use this in your other controllers to eliminate or minimise the mapping paths for individual methods